### ISSUE TITLE
Registering addons

### ISSUE THREAD

**Issue by screret** [gigaherz/JsonThings#13]
is there a way you could facilitate creating custom thing types? e.g. an api
i would like to add the ability to register some other types of things in my mod.

**Comment by screret** [comment]
also, github has a wiki tab you could use instead of a folder in the source root

**Comment by gigaherz** [comment]
I know there's a wiki tab, but it's easier for me to update the documentation as I code, in the same repository, vs trying to remember later, and failing because my memory is shit.

There's a way to register parsers for new thing types, but the timings of it may not work for other mods, I have to look into it properly. The API may not be fully complete, though, and I welcome people to try to use it and report any issues they find.

**Comment by screret** [comment]
Well then
I'll try yo set it up and will come back with patches if it doesn't work.

**Comment by hjake123** [comment]
This is possible in modern versions of Json Things -- I think this issue could be closed.

Here's an example of a parser for a custom mod thing: [Example](https://github.com/hjake123/reactive/blob/15ef3a1317ab243ec63adf5635b748c03fc619ff/src/main/java/dev/hyperlynx/reactive/integration/jsonthings/PowerParser.java)

**Comment by screret** [comment]
Yeah, I got my thing working over a year ago too. I'll close this as solved.
